ambry-rest/src/main/java/com.github.ambry.rest/NettyFrontendServerChannelInitializer.java

```diff
@@ -31,7 +31,7 @@
  * A {@link ChannelInitializer} to be used with {@link NettyServer}. Calling {@link #initChannel(SocketChannel)} adds
  * the necessary handlers to a channel's pipeline so that it may handle requests.
  */
-public class NettyServerChannelInitializer extends ChannelInitializer<SocketChannel> {
+public class NettyFrontendServerChannelInitializer extends ChannelInitializer<SocketChannel> {
```

Comment:
Recommend renaming this FrontendNettyServerChannelInitializer.
Also, do we need Server in the name, or can it be just NettyChannelInitializer?

Reply:
We will have a server Initializer.

I think we can name them as: FrontendNettyChannelInitializer,  StorageChannelInitializer.